sqlmesh/core/engine_adapter/mixins.py

```diff
@@ -184,14 +191,32 @@ def _build_view_properties_exp(
         properties: t.List[exp.Expression] = []
 
         if table_description:
-            properties.append(exp.SchemaCommentProperty(this=exp.Literal.string(table_description)))
+            properties.append(
+                exp.SchemaCommentProperty(
+                    this=exp.Literal.string(self._truncate_table_comment(table_description))
+                )
+            )
 
         properties.extend(self._table_properties_to_expressions(table_properties))
 
         if properties:
             return exp.Properties(expressions=properties)
         return None
 
+    def _truncate_table_comment(self, comment: str) -> str:
+        # iceberg tables do not have a comment length limit
+        if self.current_catalog_type == "iceberg":
+            return comment
+
+        return super()._truncate_table_comment(comment)
+
+    def _truncate_column_comment(self, comment: str) -> str:
+        # iceberg tables do not have a comment length limit
```

Comment:
Copy/paste typo: `tables` -> `columns`

It might be better to add this check to `def _truncate_comment` so it is not repeated and it has no length limit regardless of comment type.